android-sdk/android/src/main/kotlin/io/logto/sdk/android/LogtoClient.kt

```diff
@@ -72,7 +71,7 @@ open class LogtoClient(
                 refreshToken = codeTokenResponse.refreshToken
                 idToken = codeTokenResponse.idToken
 
-                completion.onComplete(null, codeTokenResponse.accessToken)
```

Comment:
Is the access token not needed at all in the completion logic?

Reply:
We get access tokens by `getAccessToken`.
Only results are needed here